text/0000-gps_support.md

```diff
@@ -0,0 +1,24 @@
+# NavSatFix (GPS) Support in Cartographer ROS
+
+## Summary
+[summary]: #summary
+
+The goal is to add NavSatFix (GPS) support to Cartographer ROS. 
```

Comment:
Add a link to http://docs.ros.org/jade/api/sensor_msgs/html/msg/NavSatFix.html

Reply:
done